syft/frameworks/torch/hook/hook.py

```diff
@@ -707,14 +707,29 @@ def get_hooked_func(hook_self, attr):
         if attr.__module__ is None:
             attr.__module__ = "torch"
 
+        cmd_name = f"{attr.__module__}.{attr.__name__}"
+
         @wraps(attr)
         def overloaded_func(*args, **kwargs):
             """
             Operate the hooking
             """
-            cmd_name = f"{attr.__module__}.{attr.__name__}"
+            try:
```

Comment:
They are faster yes because in this case the try succeeds in 99% of the cases, so it would be useless to add a if there for only 1% of the case. 
I believe this part of code is critical (it is run even when people use plain torch operations after they hooked) so speed should prevail on readability, but I must say I haven't benchmarked try / if here. I could if you think it's important